doc/reference_output.xml

```diff
@@ -623,6 +623,7 @@ F000000000000000000000000000000000000000000000000');
 				<paramdef choice="opt"><type>text </type> <parameter>geomcolumnname</parameter></paramdef>
```

Comment:
On another unrelated note, this is wrong and there is patch fixing it on Trac: https://trac.osgeo.org/postgis/ticket/4764